mycroft/messagebus/message.py

```diff
@@ -127,3 +127,20 @@ def publish(self, type, data, context=None):
             del new_context['target']
 
         return Message(type, data, context=new_context)
+
+    def utterance_remainder(self):
+        """
+        For intents get the portion not consumed by Adapt.
+
+        For example: if they say 'Turn on the family room light'
+        and there are entity matches for "turn on" and "light",
+        then it will leave behind "the family room".
+
+        Returns:
+            str: Leftover words or None if not an utterance.
+        """
+        utt = message.data["utterance", None]
```

Comment:
This is probably not correct syntax.

Reply:
Cleaned up and tested with the Wink skill (replacing its own get_remainder() implementation